interfaces/bmc/http.go

```diff
@@ -17,6 +18,15 @@ func NewDriverFromGinContext(c *gin.Context) Driver {
 		Address:  c.Param("ip"),
 		Username: c.Request.Header.Get("X-IPMI-Username"),
 		Password: c.Request.Header.Get("X-IPMI-Password"),
+		Cipher:   -1,
```

Comment:
why can't we have 0 be the unset one?

Reply:
0 is a cipher index that can be specified

Reply:
```
ID   IANA    Auth Alg        Integrity Alg   Confidentiality Alg
0    N/A     none            none            none           
1    N/A     hmac_sha1       none            none           
2    N/A     hmac_sha1       hmac_sha1_96    none           
3    N/A     hmac_sha1       hmac_sha1_96    aes_cbc_128    
...
```